src/views/Player.vue

```diff
@@ -426,33 +445,24 @@ export default {
 
         this.updateJourney("exit-fullscreen")
         this.uploadJson()
-      })
+      });
 
-      var prevTime = -1
-      player.on("timeupdate", async () => {
 
-        // Update watch time if the video is playing
-        if(this.player.playing) {
-          this.watchTime += interval_time;
-        }
-
-        // Record how many times a particular second was visited
-        var currTime = Math.trunc(this.player.currentTime);
-        if (currTime != prevTime) {
-            this.retention[currTime] += 1;
-            prevTime = currTime
-        }
```

Comment:
This piece of code is showing removed here, but it's down there if you see. Github has flagged this as removed, which is not the case